front-end/src/molecules/CollaboratorItem.css

```diff
@@ -1,7 +1,7 @@
 .CollaboratorItem {
   display: flex;
   flex-direction: row;
-  width: calc(100%-2px);
+  width: calc(100% - 2px);
```

Comment:
This is causing a scroll bar to appear
![image](https://user-images.githubusercontent.com/44079021/163731059-f2745225-fa84-4846-a8f4-eff083f2d13b.png)

Everything else looks good tho!

Reply:
Made changes here. I'm not so sure why the spacing would matter here. But I changed it so that it works. Looks the sonarcloud is failing after the change so I'm going to investigate more.

Reply:
Ok, fixed. Here's the explanation of the bug and spacing: [https://sonarcloud.io/organizations/nerdynomads/rules?open=css%3AS5362&rule_key=css%3AS5362](https://sonarcloud.io/organizations/nerdynomads/rules?open=css%3AS5362&rule_key=css%3AS5362)